### ISSUE TITLE
Google Drive link in the Readme is not working

### ISSUE THREAD

**Issue by jihyeonhur** [minar09/cloth-human-depth-estimation#1]
Hello, thank you for sharing your awesome code. 
I noticed that the Google Drive link for downloading the model in the Readme is not working. 
Could you please confirm and investigate the issue? 
Thank you.

**Comment by minar09** [comment]
Hi, the drive links are from the original repo (https://github.com/sfu-gruvi-3dv/deep_human), please refer to that for any updates. Thank you.

**Comment by jihyeonhur** [comment]
Thank you for your response. Unfortunately, I encountered the same issue with the Google Drive link in the original repository as well. When I tried to access the link, it returned a 404 error, indicating that the link is not working properly. Could you please check and investigate the issue? Thank you.

**Comment by minar09** [comment]
Yes, I also cannot access the drive links. Please contact the authors from the original repository (https://github.com/sfu-gruvi-3dv/deep_human). Thanks.

**Comment by jihyeonhur** [comment]
Oh.. I mistakenly thought that the authors from the original repository and this repository were the same. I understand now. Thank you so much for your response and clarification, I appreciate it. Thank you!
